util/src/main/java/com/google/cloud/hadoop/util/AsyncWriteChannelOptions.java

```diff
@@ -62,6 +62,10 @@ public static Builder builder() {
 
   public abstract boolean isDirectUploadEnabled();
 
+  /**
```

Comment:
Doesn't look like a right place to put this document.

Reply:
Done.